app/models/metadata_presenter/service.rb

```diff
@@ -14,20 +20,22 @@ def service_slug
   end
 
   def find_page_by_url(url)
-    pages.find { |page| strip_slash(page.url) == strip_slash(url) }
+    all_pages.find { |page| strip_slash(page.url) == strip_slash(url) }
   end
 
   def find_page_by_uuid(uuid)
-    pages.find { |page| page.uuid == uuid }
+    all_pages.find { |page| page.uuid == uuid }
   end
```

Comment:
Do we need to add unit tests for `#find_page_by_url` and `find_page_by_uuid` testing that it can find the standalone pages?